setup.md

```diff
@@ -11,6 +11,54 @@ You should have Docker Engine and Docker Compose installed. This is very fast an
 * [CentOS](https://docs.docker.com/engine/install/centos/)
 * More available at [https://docs.docker.com/engine/install/](https://docs.docker.com/engine/install/)
 
+## Recommended: CLI Installation
```

Comment:
@yaronkoren says: Need to add what the CLI is and why it's recommended. One or two sentences.

Reply:
Yeah, sure.